src/lib/converter/ts-internal.ts

```diff
@@ -0,0 +1,130 @@
+import * as ts from "typescript";
+
+/**
+ * Expose the internal TypeScript APIs that are used by TypeDoc
+ */
```

Comment:
Is there any concern about the out-datedness that could occur here? Is there a way to build it with the internal APIs exposed instead and use that?